src/components/newUIComponents/overview/TimeIncrementSelector.js

```diff
@@ -0,0 +1,50 @@
+import { Menu, MenuItem, MenuButton } from "@szhsin/react-menu";
+import { FaChevronDown } from "react-icons/fa";
+
+export default function TimeIncrementSelector({
+  timeIncrement,
+  setTimeIncrement,
+}) {
+  const timeIncrementToText = (timeIncrement) => {
```

Comment:
Time constants for hr/day/mo/yr exist in `search.js` if helpful (they're defined in ms, not strings through)